lib/guard/txi_rails_hologram.rb

```diff
@@ -0,0 +1,58 @@
+require "guard/txi_rails_hologram/version"
+require "guard"
+require "guard/plugin"
+
+module Guard
+  class TxiRailsHologram < Plugin
+    include TxiRailsHologramVersion
+    # Called when just `enter` is pressed
+    # This method should be principally used for long action like running all specs/tests/...
+    #
+    # @raise [:task_has_failed] when run_all has failed
+    # @return [Object] the task result
+    #
+    def run_all
+      system *"bundle exec rake txi_rails_hologram:build".split or throw :task_has_failed
```

Comment:
What would you think of `system * %w{bundle exec rake txi_rails_hologram:build}`?